### PR TITLE
Update Sentieon Docs

### PR DESCRIPTION
**Pull request by edmundmiller**
Waiting until we get #1380 up and running.

### CODE REVIEW

**Review by FriederikeHanssen — DISMISSED** [review]
Sentieon server is up and running so I think we can merge this. 

I have only tested out adding a env config

```
env {
    SENTIEON_LICENSE = 'IP:PORT'
}
```

without the secrets, but this works great

**Comment by maxulysse** [comment]
Can you update CHANGELOG?

**Comment by edmundmiller** [comment]
@maxulysse As you wish!

**Comment by FriederikeHanssen** [comment]
@edmundmiller is this ready to go?

File: docs/usage.md [hunk 1]
```diff
@@ -1104,12 +1104,20 @@ Sentieon supply license in the form of a string-value (a url) or a file. It shou
...
+:::
+
 If a license file is supplied, then the nextflow secret should be set like this:
 
 ```bash
 nextflow secrets set SENTIEON_LICENSE_BASE64 \$(cat <sentieon_license_file.lic> | base64 -w 0)
 ```
 
+:::note
+If you're looking for documentation on how the nf-core Sentieon GitHub Actions and Sentieon License Server are set up: [Here be dragons.](https://github.com/nf-core/ops/blob/main/pulumi/aws_license-server/README.md)
```
Comment (maxulysse) [inline docs/usage.md:1118]:
```suggestion
If you're looking for documentation on how the nf-core Sentieon GitHub Actions and Sentieon License Server are set up: [Here be dragons.](https://github.com/nf-core/ops/blob/main/pulumi/sentieon_license_server/README.md)
```

**Review by maxulysse — APPROVED** [review]

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**